core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -98,14 +105,20 @@ object NodepoolStatus {
     override def toString: String = "RECONCILING"
   }
 
-  case object Stopping extends NodepoolStatus {
+  //in kubernetes statuses, they label "deleting' resources as 'stopping'
```

Comment:
That sounds confusing. Should we map it to `stopping` internally?

Reply:
Yeah, that's essentially what I've done. If google returns a status of "STOPPING", then it really means "Deleting" as far as we are concerned.

In other words, the way we serializable a value of "STOPPING" from google will be as our "Deleting" case object

Reply:
Does that work with 

```scala
case KubernetesClusterStatus.Deleting => Deleting
```

in `fromKubernetesClusterStatus()`? It felt like it should be

```scala
case KubernetesClusterStatus.Deleting => Stopping
```

🤔

Reply:
fromKubernetesClusterStatus maps a KubernetesClusterStatus to a RuntimeStatus

So what `case KubernetesClusterStatus.Deleting => Deleting` is saying is that `KubernetesClusterStatus.Deleting` is analogous to `RuntimeStatus.Deleting`. In other words the `STOPPIN` status that google returns maps to `KubernetesClusterStatus.Deleting` which maps to `RuntimeStatus.Deleting`